vaadin-combo-box-overlay.html

```diff
@@ -0,0 +1,213 @@
+<!--
+@element vaadin-combo-box-overlay
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../iron-list/iron-list.html">
+<link rel="import" href="vaadin-overlay-behavior.html">
+
+<dom-module id="vaadin-combo-box-overlay">
+  <style>
+    :host {
+      position: absolute;
+      z-index: 100;
+      @apply(--shadow-elevation-2dp);
+      background: #fff;
+      border-radius: 0 0 2px 2px;
+    }
+
+    #scroller {
+      overflow: auto;
+      max-height: 65vh;
```

Comment:
This value only works okay if `<meta name="viewport" content="width=device-width, initial-scale=1" />` is used. On a larger viewport size the dropdown will also be larger, and when for instance iOS zooms in on the focused input, the dropdown will not fit inside the zoomed viewport.

Doesn’t block this PR, but something to consider in the future, if we can somehow improve the “guesstimate” of the dropdown height.